{"uuid": "aa9c31df-32f1-4561-809a-c7ff3f8c3f7a", "children": ["ca9618ff-ef4c-40af-adb7-0c288611e254"], "befores": [{"name": "app", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: 'geckodriver' executable needs to be in PATH. \n\n", "trace": "  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\_pytest\\fixtures.py\", line 1117, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\_pytest\\fixtures.py\", line 924, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\needl\\PycharmProjects\\python\\conftest.py\", line 30, in app\n    fixture = App(browser=browser, base_url=web_config[\"baseUrl\"])\n  File \"C:\\Users\\needl\\PycharmProjects\\python\\fixture\\app.py\", line 11, in __init__\n    self.wd = webdriver.Firefox()\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\selenium\\webdriver\\firefox\\webdriver.py\", line 164, in __init__\n    self.service.start()\n  File \"c:\\users\\needl\\pycharmprojects\\python\\env\\lib\\site-packages\\selenium\\webdriver\\common\\service.py\", line 83, in start\n    os.path.basename(self.path), self.start_error_message)\n"}, "start": 1597352774788, "stop": 1597352774792}], "start": 1597352774788, "stop": 1597352774863}